internal/cli/command/cluster/integratedservice/services/backup/enable.go

```diff
@@ -223,12 +223,29 @@ func buildEnableRequestInteractively(banzaiCli cli.Cli) (pipeline.EnableArkReque
 		return pipeline.EnableArkRequest{}, errors.WrapIf(err, "failed to get bucket")
 	}
 
+	useClusterSecret := false
+	if options.ClusterCloud() == amazonType {
+		err = input.DoQuestions([]input.QuestionMaker{
+			input.QuestionConfirm{
+					QuestionBase: input.QuestionBase{
+						Message: "Deploy cluster secret to give access for Velero to make volume snapshots",
+						Help:    "In case you are not deploying cluster secret you must add snapshot permissions to your instance profile."},
```

Comment:
```suggestion
						Help:    "This option deploys the cloud provider secret used for creating this cluster to the cluster itself. In case you are not deploying cluster secret you must add snapshot permissions to your instance profile."},
```